pretixdroid/app/src/main/java/eu/pretix/pretixdroid/ui/EventinfoActivity.java

```diff
@@ -0,0 +1,338 @@
+package eu.pretix.pretixdroid.ui;
+
+import android.os.AsyncTask;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.LinkedList;
+
+import eu.pretix.pretixdroid.AppConfig;
+import eu.pretix.pretixdroid.R;
+import eu.pretix.pretixdroid.net.api.ApiException;
+import eu.pretix.pretixdroid.net.api.PretixApi;
+
+/**
+ * This class is the activity for the Eventinfo page to let the user see statistics about their
+ * event.
+ * @author jfwiebe
+ */
+public class EventinfoActivity extends AppCompatActivity {
+    private AppConfig config;
+    private PretixApi api;
+
+    private ListView mListView;
+    private EventItemAdapter mAdapter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_eventinfo);
+
+        this.mListView = (ListView)findViewById(R.id.eventinfo_list);
+        this.mAdapter = new EventItemAdapter(getBaseContext());
+        this.mListView.setAdapter(this.mAdapter);
+
+        this.config = new AppConfig(this);
+        this.api = PretixApi.fromConfig(config);
+
+        new StatusTask().execute();
+    }
+
+    public class StatusTask extends AsyncTask<String, Integer, JSONObject> {
+
+        /**
+         * exexutes an asyncron request to obtain status information from the pretix instance
+         * @param params are ignored
+         * @return the associated json object recieved from the pretix-endpoint or
+         * @return null if the request was not successful
+         */
+        @Override
+        protected JSONObject doInBackground(String... params) {
+            try {
+                return EventinfoActivity.this.api.status();
+            } catch (ApiException e) {
+                e.printStackTrace();
+            }
+            return null;
+        }
+
+        /**
+         * it parses the answer of the pretix endpoint into objects and adds them to the list
+         * @param result the answer of the pretix status endpoint
+         */
+        @Override
+        protected void onPostExecute(JSONObject result) {
+            EventItemAdapter eia = EventinfoActivity.this.mAdapter;
+            try {
+                EventCardItem ici = new EventCardItem(result);
+                eia.addItem(ici);
```

Comment:
just in case this runs twice for some reason, shouldn't it clear the adapter first?